package.json

```diff
@@ -165,7 +165,7 @@
     "cozy-authentication": "2.1.0",
     "cozy-bar": "^8.3.0",
     "cozy-ci": "0.4.1",
-    "cozy-client": "24.1.0",
+    "cozy-client": "27.4.0",
```

Comment:
this is not a "chore" commit. Since the code is present in the bundle...

Reply:
You're not concerned by the breaking change? 

Specially this one: https://github.com/cozy/cozy-client/blob/master/CHANGELOG.md#2500-2021-10-12 

There is a few places where banks use save() https://github.com/cozy/cozy-banks/search?q=save%28 .

Reply:
Oops, thx

Reply:
Can you also upgrade cozy-stack-client and cozy-pouch-link please?

Reply:
It's upgraded, and it seems ok about `_type` in save.